web_dashboard_tile/models/tile_tile.py

```diff
@@ -46,70 +84,113 @@ def _context_today():
-
     active = fields.Boolean(
         compute='_compute_active',
         search='_search_active',
         readonly=True)
 
+    # Primary Value
+    primary_function = fields.Selection(
+        FIELD_FUNCTION_SELECTION,
+        string='Function',
+        default='count')
+    primary_field_id = fields.Many2one(
+        'ir.model.fields',
+        string='Field',
+        domain="[('model_id', '=', model_id),"
+               " ('ttype', 'in', ['float', 'int'])]")
+    primary_format = fields.Char(
+        string='Format',
+        help='Python Format String valid with str.format()')
+    primary_value = fields.Char(
+        string='Value',
+        compute='_compute_data')
+    primary_helper = fields.Char(
+        string='Helper',
+        compute='_compute_helper')
+
+    # Secondary Value
+    secondary_function = fields.Selection(
+        FIELD_FUNCTION_SELECTION,
+        string='Secondary Function')
+    secondary_field_id = fields.Many2one(
+        'ir.model.fields',
+        string='Secondary Field',
+        domain="[('model_id', '=', model_id),"
+               " ('ttype', 'in', ['float', 'int'])]")
+    secondary_format = fields.Char(
+        string='Secondary Format',
+        help='Python Format String valid with str.format()')
+    secondary_value = fields.Char(
+        string='Secondary Value',
+        compute='_compute_data')
+    secondary_helper = fields.Char(
+        string='Secondary Helper',
+        compute='_compute_helper')
+
+    error = fields.Char(
+        string='Error Details',
+        compute='_compute_data')
+
     @api.one
     def _compute_data(self):
-        self.count = 0
-        self.computed_value = 0
-        if self.active:
-            # Compute count item
+        if not self.active:
+            return
+        try:
             model = self.env[self.model_id.model]
             eval_context = self._get_eval_context()
-            self.count = model.search_count(eval(self.domain, eval_context))
-            # Compute datas for field_id depending of field_function
-            if self.field_function and self.field_id and self.count != 0:
-                records = model.search(eval(self.domain, eval_context))
-                vals = [x[self.field_id.name] for x in records]
-                if self.field_function == 'min':
-                    self.computed_value = min(vals)
-                elif self.field_function == 'max':
-                    self.computed_value = max(vals)
-                elif self.field_function == 'sum':
-                    self.computed_value = sum(vals)
-                elif self.field_function == 'avg':
-                    self.computed_value = sum(vals) / len(vals)
-                elif self.field_function == 'median':
-                    self.computed_value = self.median(vals)
+            domain = self.domain or '[]'
+            count = model.search_count(eval(domain, eval_context))
+            if any([
+                self.primary_function and
```

Comment:
We want only the cases where `***_function` is `True`ish **and** not `count`.

The following comparisson should result as following
```
False -> False
'count' -> False
'any_other_func' -> True
```

If the comparisson is only `!= 'count'` then it would be as follows:
```
False -> True
'count' -> False
'any_other_func' -> True
```